playbook-website/app/javascript/components/VisualGuidelines/Examples/Spacing.tsx

```diff
@@ -30,11 +30,12 @@ const PROPNAMES = [
   'paddingY' ]
 
 const TOKENS = {
-  'Extra Small': 'space_xs',
+  'XX Small': 'space_xxs',
+  'X Small': 'space_xs',
   'Small': 'space_sm',
   'Medium': 'space_md',
   'Large': 'space_lg',
-  'Extra Large': 'space_xl',
+  'X Large': 'space_xl',
```

Comment:
I personally don't have an issue with using `X` here instead of spelling out `Extra` (esp. twice lol).